test/slow/git-spawn-test.ts

```diff
@@ -12,13 +12,15 @@ const maximumStringSize = 268435441
 function bufferOutput(process: ChildProcess, failPromiseWhenLengthExceeded: boolean = true) {
   return new Promise<string>((resolve, reject) => {
     const stdout: Array<Buffer> = []
-    process.stdout.on('data', chunk => {
-      if (chunk instanceof Buffer) {
-        stdout.push(chunk)
-      } else {
-        stdout.push(Buffer.from(chunk))
-      }
-    })
+    if (process.stdout) {
+      process.stdout.on('data', chunk => {
+        if (chunk instanceof Buffer) {
+          stdout.push(chunk)
+        } else {
+          stdout.push(Buffer.from(chunk))
+        }
+      })
+    }
```

Comment:
Can't we use optional chaining here? 🤔 
```suggestion
    process.stdout?.on('data', chunk => {
      if (chunk instanceof Buffer) {
        stdout.push(chunk)
      } else {
        stdout.push(Buffer.from(chunk))
      }
    })
```

Reply:
Not yet, the version of prettier that we use in dugite didn't support which is why I did it this way (sorry, should have made a comment or at least a review comment about it) it but I'm planning on bumping prettier in a separate PR (this one got noisy enough)